lib/puppet/functions/cd4pe_deployments/get_pipeline_trigger_event.rb

```diff
@@ -24,18 +24,16 @@
   end
 
   def get_pipeline_trigger_event(repo_name, pipeline_id, commit_sha)
-    client = PuppetX::Puppetlabs::CD4PEClient.new
+    results = call_function('cd4pe_deployments::get_pipeline_trigger_events', repo_name, pipeline_id, commit_sha)
```

Comment:
I see the method already exists, but do we want to expose this information as "trigger events" to our customers? I wonder if we might want to label this method as something more self-descriptive, like `getPipelineEvent` or `getTimelineEvent`. Just throwing the idea out there, not sure it's worth the breaking change.

Reply:
I'd prefer to deprecate this function, but we can't introduce a backwards breaking change until 4.0. Not sure how our deprecation process works. And maybe we don't need to worry about it since it's still in beta?

Reply:
I think "pipeline run events" makes more sense, but we might want to just stick with "trigger events" for the sake of consistency.